apps/src/templates/manageStudents/Table/UsStateColumn/BulkSetModal/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react';
+import {ControlLabel, FormControl, FormGroup} from 'react-bootstrap';
+import {connect, useSelector} from 'react-redux';
+
+import {Heading4} from '@cdo/apps/componentLibrary/typography';
+import {STATE_CODES} from '@cdo/apps/geographyConstants';
+import StylizedBaseDialog from '@cdo/apps/sharedComponents/StylizedBaseDialog';
```

Comment:
If it is not too complicated, should we be using AccesibleDialog instead?

Reply:
Anything for you, Mario!

Reply:
[P20-1124: Use AccessibleDialog component for modal](https://github.com/code-dot-org/code-dot-org/pull/60873/commits/7cdc6f41a43d414db8492d18d062b4b2eebbcf0a)

<img  alt="modal-accessibility" src="https://github.com/user-attachments/assets/7aabf772-2f09-48b9-b894-40a8c11a0d09">